blocks/browse/da-browse/da-browse.css

```diff
@@ -19,6 +19,62 @@ input, button {
   font-family: var(--body-font-family);
 }
 
+/* Tab List */
+.da-browse-tab-header {
+  display: flex;
+  gap: 18px;
+  margin-bottom: 24px;
+}
+
+.da-browse-tab-header button {
+  position: relative;
+  display: block;
+  background: none;
+  border: none;
+  padding: 0 0 6px;
+  margin: 0;
+  box-sizing: border-box;
+  color: #8f8f8f;
```

Comment:
Would it be an idea to start using variables for colors so we can easily change the color scheme going forward if we want to?